src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -109,4 +109,21 @@ public void addMessage(Message message) {
   public void setMessages(List<Message> messages) {
     this.messages = messages;
   }
+
+  public int getNumOfMessages(){
+    return messages.size();
+  }
+
+  public int getMessagesFromAuthorSize(UUID authorId) {
```

Comment:
This method can just call `getMessagesFromAuthor`, then return its size.
Or you could just get rid of this method, and have the caller call `getMessagesFromAuthor('name').size()`?